### ISSUE TITLE
Connect-a-thon: No print dialog appears if you buy a label when certain rates are selected

### ISSUE THREAD

**Issue by allendav** [Automattic/woocommerce-connect-client#574]
Observed in Connect-a-thon. Maybe 0.75 lbs is too heavy? If you click on Buy & Print when the First-Class Mail - Stamped Letter rate is selected, no print dialog results. (If you pick Priority Mail 1-Day it works)

<img width="851" alt="screen shot 2016-09-27 at 6 24 12 pm" src="https://cloud.githubusercontent.com/assets/1595739/18897507/d664cb2a-84df-11e6-9d22-9b4b07b977a3.png">

**Comment by allendav** [comment]
Also observed for Priority Mail Express 1-Day

**Comment by allendav** [comment]
Putting an estimate of 1 until we have a better sense of what it happening here - possibly 500s coming from the server for priority mail

**Comment by DanReyLop** [comment]
I've investigated the issue. EasyPost uses internally different APIs for ["regular" mail](https://www.usps.com/business/web-tools-apis/delivery-confirmation-domestic-shipping-label-api.htm) and for ["Priority Express" mail](https://www.usps.com/business/web-tools-apis/priority-mail-express-api.htm).

The normal API doesn't require the parameter `FromPhone`, but the Express API does (10-digit number, without punctuation). I managed to purchase a Priority Mail Express label hard-coding a phone number in the server.

The solution is to add a `Phone number` field to the `Origin address` form. Validating the phone should be done both in the client and the server. We would also need minor changes in the server to pass that phone number to EasyPost.

This will make the phone field mandatory even for services that don't require it, but it's the simplest solution, since at the moment of entering the Origin Address we don't know which rate the merchant is going to choose.

@allendav: I would give this issue (combined client and server) an estimate of `3` (I would give it a `2` but validating a phone number is never trivial). I will leave it as a "stretch goal" for this sprint. Or, if someone else wants to take this, I think the solution is pretty straightforward.
